package.json

```diff
@@ -19,13 +19,28 @@
   "license": "ISC",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/blackbaud/skyux-sample.git"
+    "url": "git+https://github.com/blackbaud/skyux-list-builder-view-grids.git"
   },
   "bugs": {
-    "url": "https://github.com/blackbaud/skyux-sample/issues"
+    "url": "https://github.com/blackbaud/skyux-list-builder-view-grids/issues"
+  },
+  "homepage": "https://github.com/blackbaud/skyux-list-builder-view-grids#readme",
+  "peerDependencies": {
+    "@angular/common": "^4.3.6",
+    "@angular/core": "^4.3.6",
+    "@skyux/animations": ">=3.0.0-rc.0",
+    "@skyux/forms": ">=3.0.0-rc.0",
+    "@skyux/grids": "3.0.0-alpha.1",
+    "@skyux/indicators": ">=3.0.0-rc.1",
+    "@skyux/layout": ">=3.0.0-rc.0",
+    "@skyux/list-builder": "3.0.0-alpha.0",
+    "@skyux/lists": ">=3.0.0-rc.0",
+    "@skyux/lookup": ">=3.0.0-rc.0",
```

Comment:
I know I've said differently in the past, but peer dependencies should only use the caret `^` for now, to ensure that only pre-release versions are accepted.

Reply:
done.